Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -49,3 +49,40 @@ If any setting is unspecified, a suitable default will be used automatically.
 ### Spec
 
 - `blockPoolName`: The metadata name of the CephBlockPool CR where the rados namespace will be created.
+
+
+## Creating a Storage Class
+
+Once the RADOS namespace is created, an RBD-based StorageClass can be created to
+create PVs in this RADOS namespace. For this purpose, the `clusterID` value from the
+CephBlockPoolRadosNamespace status needs to be put into the `clusterID` field of the StorageClass
+spec. 
+cluster.
+
+Extract the clusterID from the CephBlockPoolRadosNamespace CR:
+
+ ```shell
+ $ kubectl -n rook-ceph get cephblockpoolradosnamespace -o yaml
```

Comment:
can we provide the direct cmd
```
$KUBECTL -n "$NAMESPACE" get cephblockpoolradosnamespace.ceph.rook.io/"$RADOS_NAMESPACE" -o jsonpath='{.status.info.clusterID}'
```

Reply:
@parth-gr  wrote:

> can we provide the direct cmd
> 
> ```
> $KUBECTL -n "$NAMESPACE" get cephblockpoolradosnamespace.ceph.rook.io/"$RADOS_NAMESPACE" -o jsonpath='{.status.info.clusterID}'
> ```

good idea! I will play with it a bit but it is currently unclear to me how/where  the variables  `$KUBECTL`, `$NAMESPACE`, etc  are expected to be set.

Reply:
you can directly use `kubectl` and namespace as `rook-ceph`

Reply:
it works with this script https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh#L128

Reply:
lets catch up and check, if it doesn't works

Reply:
@parth-gr  more details about the error I am facing:


```shell
$ kubectl  -n rook-ceph get cephblockpoolradosnamespaces.ceph.rook.io/namespace-a  -o jsonpath='{status.info.ClusterID}'

error: error executing jsonpath "{status.info.ClusterID}": Error executing template: unrecognized identifier status. Printing more information for debugging the template:
	template was:
		{status.info.ClusterID}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"ceph.rook.io/v1", "kind":"CephBlockPoolRadosNamespace", "metadata":map[string]interface {}{"creationTimestamp":"2024-05-07T14:27:11Z", "finalizers":[]interface {}{"cephblockpoolradosnamespace.ceph.rook.io"}, "generation":1, "managedFields":[]interface {}{map[string]interface {}{"apiVersion":"ceph.rook.io/v1", "fieldsType":"FieldsV1", "fieldsV1":map[string]interface {}{"f:spec":map[string]interface {}{".":map[string]interface {}{}, "f:blockPoolName":map[string]interface {}{}}}, "manager":"kubectl-create", "operation":"Update", "time":"2024-05-07T14:27:11Z"}, map[string]interface {}{"apiVersion":"ceph.rook.io/v1", "fieldsType":"FieldsV1", "fieldsV1":map[string]interface {}{"f:metadata":map[string]interface {}{"f:finalizers":map[string]interface {}{".":map[string]interface {}{}, "v:\"cephblockpoolradosnamespace.ceph.rook.io\"":map[string]interface {}{}}}}, "manager":"rook", "operation":"Update", "time":"2024-05-07T14:27:11Z"}, map[string]interface {}{"apiVersion":"ceph.rook.io/v1", "fieldsType":"FieldsV1", "fieldsV1":map[string]interface {}{"f:status":map[string]interface {}{".":map[string]interface {}{}, "f:info":map[string]interface {}{".":map[string]interface {}{}, "f:clusterID":map[string]interface {}{}}, "f:phase":map[string]interface {}{}}}, "manager":"rook", "operation":"Update", "subresource":"status", "time":"2024-05-08T12:34:52Z"}}, "name":"namespace-a", "namespace":"rook-ceph", "resourceVersion":"28365", "uid":"fcae1125-b349-4260-b9da-d46f795128ce"}, "spec":map[string]interface {}{"blockPoolName":"replicapool"}, "status":map[string]interface {}{"info":map[string]interface {}{"clusterID":"80fc4f4bacc064be641633e6ed25ba7e"}, "phase":"Failure"}}
$
```
```

Reply:
it should be a small `c` not the capital one for `clusterID`

Reply:
tesyed and done.

Reply:
> @parth-gr , I tried this:
> 
> ```
> kubectl  -n rook-ceph get cephblockpoolradosnamespaces.ceph.rook.io/namespace-a  -o jsonpath='{status.info.ClusterID}'
> ```
> 
> but only got garbage as output.
> 
> I also tried to find a command to use `jsonpath` to directly get the namespace name (in this case `namespace-a`, but no success yet.



@parth-gr  now I managed to produce a working command and updated the doc accordingly :)